x-pack/plugins/lens/public/indexpattern_service/loader.ts

```diff
@@ -298,11 +298,14 @@ export async function syncExistingFields({
   currentIndexPatternTitle: string;
   dslQuery: object;
   onNoData?: () => void;
+  core: Pick<CoreStart, 'http' | 'notifications' | 'uiSettings'>;
+  data: DataPublicPluginStart;
+  dataViews: DataViewsContract;
 }) {
   const { indexPatternList } = requestOptions;
   const newExistingFields = { ...existingFields };
 
-  const { result, status } = await refreshExistingFields(requestOptions);
+  const { result, status } = await refreshExistingFields({ ...requestOptions });
```

Comment:
nit: Why this spread? Seems like it's not necessary

Reply:
Looks like leftover, removed spread.